supported modules on AA (#2247)
RedisLabs/redislabs-docs: content/rs/release-notes/rs-6-2-18.md

## Features and enhancements

## General Availability of Active-Active databases with RedisJSON

> Index, query, and full-text search of nested JSON documents are now available on Active-Active  databases by using the combination of RedisJSON with RediSearch.

Reviewer: ```suggestion
Active-Active databases now support Index, query, and full-text search of nested JSON documents when combining RedisJSON with RediSearch.
```

> The capability of indexing, query and full-text search of nested JSON documents are now also GA on Active-Active (achieved by the combination of RedisJSON with RediSearch).

Reviewer 2: ```suggestion
Index, query, and full-text search of nested JSON documents are now available on Active-Active  databases by using the combination of RedisJSON with RediSearch.
```

### Active-Active support for MEMORY USAGE command

Redis Enterprise Active-Active databases now support the [MEMORY USAGE](https://redis.io/commands/memory-usage/) command, which simplifies troubleshooting and lets applications detect anomalous behavior and dangerous trends.

> Find out more about [`Active-Active with RedisJSON`]({{<relref "//modules/redisjson/active-active">}})

Reviewer: ```suggestion
To learn more, see  [`Active-Active with RedisJSON`]({{<relref "//modules/redisjson/active-active">}}).
```
